src/components/common/action-button/index.js

```diff
@@ -3,34 +3,26 @@ import { jsx } from '@emotion/core'
 import React from 'react'
 import * as ReactRouter from 'react-router-dom'
 
+import Button from 'components/common/button'
+import WALLET_BUTTON from 'constants/wallet_button.js'
+
 import style from './index.style.js'
 
 class ActionButton extends React.Component {
   render() {
+    const { type } = this.props
     return (
-      <button
-        css={style.base}
+      <Button
+        css={[style.base, { backgroundColor: WALLET_BUTTON.COLOR[type] }]}
```

Comment:
Or even better, how about we keep component constants with components them-self?
Like `components/onboarding/get-wallet/constants.js`, `components/shopping-cart/constants.js`?

We will have constants in 2 places, but these are going to be presentational constants only. 

Potentially we could end up with both `constants/store` and `components/store/constants` but I think it wouldn't be too bad since presentational constants will not be used outside `components/*` and we will import them with:

```js
import STORE_SCENE from './constants'
import STORE from './constants/store'
```

wdyt?

@pycckuu?

Reply:
I'd like to have all constants in the constant folder.  Less thinking for future Aaron that way.

Reply:
this has been addressed in commit https://github.com/ChingStore/ching/pull/45/commits/f5f28ece84d3c72b92f79e11aabc904f22e05f0b